src/Humanizer.Tests.Shared/Localisation/de/NumberToWordsTests.cs

```diff
@@ -7,7 +7,7 @@ public class NumberToWordsTests
     {
         [Theory]
         [InlineData(0, "null")]
-        [InlineData(1, "eins")]
+        [InlineData(1, "ein")]
```

Comment:
Would this make `1.ToWords() == "ein"`? I agree that ie. "1 Haus" should be "Ein Haus", but when displayed ie. in a table, `1` should be [`"Eins"`](https://de.wikipedia.org/wiki/Eins), unless I'm missing something.

Reply:
Yes you're right, this is in a sense a "breaking" change. However, since `ToQuantity` uses `ToWords` when called with `ShowQuantityAs.Numeric` the current output of `"eins"` is grammatically wrong in any case because it's the numbers name when counting.
As far as I see there is no possibilty to add this corner case with the current architecture of Humanizer.

I don't know about other languages but for German we would need for possible outputs for the number `1`:
* `"ein"` for `GrammaticalGender.Masculine`,
* `"eine"` for `GrammaticalGender.Feminine`,
* `"ein"` for `GrammaticalGender.Neuter` and
* `"eins"` when you just need the number's name (something like `GrammaticalGender.None` if you will)

I'm aware this is a limitation but as said I think from the grammatical perspective this implementation is more correct since the method always gets a `GrammaticalGender` provided (by default `GrammaticalGender.Masculine`) and in this case the output `"eins"` is wrong.

Reply:
Yup I guess that's a limitation in Humanizer right now. I just wanted to make sure this change was made deliberately. Other than that, I think this PR is a great improvement. 👍